src/endpoints/msigs/domain/models/msig-aggregate-record.ts

```diff
@@ -0,0 +1,126 @@
+import { AssetRawMapper, Pair } from '@alien-worlds/aw-antelope';
+import * as MSIGWorldsCommon from '@alien-worlds/aw-contract-msig-worlds';
+import {
+  removeUndefinedProperties,
+  UnknownObject,
+} from '@alien-worlds/aw-core';
+import { camelCase } from 'change-case';
+
+import { Proposal } from '../entities/proposals';
+import { UnPackedTransaction } from './unpacked-transaction';
+
+/**
+ * The `MSIGAggregateRecord` class is responsible for creating an aggregated record that contains information about a MSIGProposal
+ * and its related data.
+ */
+export class MSIGAggregateRecord {
```

Comment:
I would consider it as an entity not a model. this is not bad but the model should be simple and we are thinking to get rid of the models in the domain (in that form).